### PR TITLE
Editor style (LBSD-291)

### PR DESCRIPTION
**Pull request by vied12**
1) edit part (left) should be with fixed width, timeline variable width (now its fifty fifty)
2) there should be a 3px gray line in the middle as vertical separator

### CODE REVIEW

File: app/scripts/ng-sir-trevor-blocks.js [hunk 1]
```diff
@@ -30,10 +30,10 @@ define([
 
                 editorHTML: function() {
                     var template = _.template([
-                        '<div class="st-required st-text-block st-quote-block quote-input" ',
+                        '<div class="st-required st-quote-block quote-input" ',
```
Comment (nistormihai) [inline app/scripts/ng-sir-trevor-blocks.js]: if st-text-block class is removed why not from css aswell?

Comment (vied12) [inline app/scripts/ng-sir-trevor-blocks.js]: we have a class `st-<block_type>-block` for each block type.

`st-text-block` is still used for the Text block

File: app/scripts/liveblog-edit/styles/liveblog-edit.less [hunk 2]
```diff
@@ -152,12 +157,33 @@
...
+                                    .btn;
+                                &:before {
+                                    text-transform: uppercase;
+                                    font-family: @baseFontFamily; // remove the sir-trevor icon behavior
+                                } 
+                            }
+                            .st-block-ui-btn--confirm-delete {.btn-danger; &:before {content: "delete"; }}
+                            .st-block-ui-btn--deny-delete {
+                                .buttonStates(#DBDBDB, @black); // needed to give a no transparent background in order to hide the cross
+                                &:before {content: "cancel"; }
```
Comment (vied12) [inline app/scripts/liveblog-edit/styles/liveblog-edit.less]: @nistormihai I have have comment to do on my own commit :)
Here I am setting a string in English in the css.
But I didn't find another way to do it in accordance with the given design and _sir-Trevor_.
For a later translation, IMO we will have three options :
- return to the icon buttons
- fork sir-trevor
- make our own editor

Comment (nistormihai) [inline app/scripts/liveblog-edit/styles/liveblog-edit.less]: sir-trevor has it's own translation system ... can't we use that?

Comment (vied12) [inline app/scripts/liveblog-edit/styles/liveblog-edit.less]: not for this text. Originally there is no text for this button, only an icon.

**Comment by nistormihai** [comment]
LGTM

File: app/scripts/liveblog-edit/styles/liveblog-edit.less (hunk 2 above)
Comment (nistormihai) [inline app/scripts/liveblog-edit/styles/liveblog-edit.less]: ok we will see for translation later.

**Pull request merged**